internal/msg/errormsg.go

```diff
@@ -184,8 +184,8 @@ const (
 	UnableToFetchFile = "Unable to fetch file '%s'"
 	// UnableToUnmarshallFile indicates a failure to unmarshall a file
 	UnableToUnmarshallFile = "Unable to unmarshall file '%s'"
-	// SkippingSmartRetries indicates that the full suite will be run
-	SkippingSmartRetries = "Skipping SmartRetry, retrying complete suite"
+	// SkippingSmartRetries implies that all tests will be retried.
+	SkippingSmartRetries = "SmartRetry is being skipped, and all tests from the previous job are being retried"
```

Comment:
Simplifying the tense (no need for present continuous; emphasizing the ongoing nature of this event is not what's important):
```suggestion
	SkippingSmartRetries = "Skipping SmartRetry. Retrying all tests from the previous job."
```